ci/ci/build.py

```diff
@@ -373,15 +390,16 @@ def build(self, batch, code, scope):
 
 export BUILDKITD_FLAGS='--oci-worker-no-process-sandbox --oci-worker-snapshotter=overlayfs'
 export BUILDCTL_CONNECT_RETRIES_MAX=100 # https://github.com/moby/buildkit/issues/1423
-retry buildctl-daemonless.sh \
-     build \
-     --frontend dockerfile.v0 \
-     --local context={shq(context)} \
-     --local dockerfile=/home/user \
-     --output 'type=image,"name={shq(self.image)},{shq(self.cache_repository)}",push=true' \
-     --export-cache type=inline \
-     --import-cache type=registry,ref={shq(self.cache_repository)} \
-     --import-cache type=registry,ref={shq(self.main_branch_cache_repository)} \
+retry buildctl-daemonless.sh \\
+     build \\
+     --frontend dockerfile.v0 \\
+     --local context={shq(context)} \\
```

Comment:
Every line here except for the added `{build_args_str}` line is just changing `\` to `\\` so it renders nicely with line breaks in the job page.